build.gradle

```diff
@@ -31,7 +31,6 @@ dependencies {
 	shadow group: 'net.kyori', name: 'adventure-text-serializer-bungeecord', version: '4.3.2'
 
 	implementation group: 'io.papermc.paper', name: 'paper-api', version: '1.21-R0.1-SNAPSHOT'
-	implementation group: 'org.eclipse.jdt', name: 'org.eclipse.jdt.annotation', version: '2.2.700'
 
 	// Comes from Minecraft including Guava or Gson
```

Comment:
Can you remove this line and the empty line above to make it the same as master and patch